src/screens/token/components/default-token/default-token.tsx

```diff
@@ -118,6 +119,26 @@ export class DefaultTokenScreenComponent extends React.Component<
                         >
                             {translate('App.labels.receive')}
                         </Button>
+                        {/* TODO: maybe find a better way to handle this */}
+                        {this.props.account.blockchain === Blockchain.ZILLIQA &&
+                            token.symbol.toLowerCase() === 'gzil' && (
```

Comment:
If we want to enable other swap tokens, this will be a blocker. 
Maybe is better to add here a list of supported tokens for swap by using firebase remote config.